src/Parser/Druid/Balance/Modules/Spells/MoonSpells.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+
+class MoonSpells extends Analyzer {
+
+  get availableCasts() {
+    //const hasRM = combatant.hasBack(ITEMS.RADIANT_MOONLIGHT.id);
+    const hasRM = false;
+    const cooldown = hasRM ? 10 : 15;
+    const totalFromCD = Math.floor(this.owner.fightDuration / 1000 / cooldown);
+    const totalFromPull = 3;
+    return totalFromCD + totalFromPull;
+  }
+
+  suggestions(when) {
+    const abilityTracker = this.owner.modules.abilityTracker;    
+    const casted = abilityTracker.getAbility(SPELLS.NEW_MOON.id).casts 
```

Comment:
Seeing as this is repeated in the `statistic()` as well, you could refactor it into a getter and access it with `this.totalCasts`.
```jsx
get totalCasts() {
   return this.abilityTracker.getAbility(SPELLS.NEW_MOON.id).casts + this.abilityTracker.getAbility(SPELLS.HALF_MOON.id).casts + this.abilityTracker.getAbility(SPELLS.FULL_MOON.id).casts;
}
```